lib/ui/src/modules/ui/components/header.js

```diff
@@ -33,44 +33,58 @@ const shortcutIconStyle = {
   cursor: 'pointer',
   display: 'inlineBlock',
   padding: 0,
-  margin: '0 0 0 5px',
+  margin: isMobileDevice ? '0 15px' : '0 0 0 5px',
   backgroundColor: 'inherit',
   outline: 0,
   width: 30,
   flexShrink: 0,
+});
+
+const burgerIconStyle = {
+  ...iconStyle(true),
+  paddingBottom: 2,
 };
 
-const linkStyle = {
+const linkStyle = isMobileDevice => ({
   textDecoration: 'none',
   flexGrow: 1,
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
-  border: '1px solid rgb(193, 193, 193)',
+  border: isMobileDevice ? 'none' : '1px solid rgb(193, 193, 193)',
   borderRadius: 2,
-};
+});
 
-const Header = ({ openShortcutsHelp, name, url }) => (
-  <div style={wrapperStyle}>
-    <a style={linkStyle} href={url} target="_blank" rel="noopener noreferrer">
+const Header = ({ openShortcutsHelp, onBurgerButtonClick, name, url, isMobileDevice }) => (
+  <div style={wrapperStyle(isMobileDevice)}>
+    {isMobileDevice && (
+      <button style={burgerIconStyle} onClick={onBurgerButtonClick}>
```

Comment:
Why don't just show this button always? I don't see any cons to it in a web mode, and having it in both modes will reduce complexity.

Reply:
for me, when I see a ☰ button on mobile websites, I expect it to "always be there for me" :) 
in this particular case, if we leave it on Desktop version and have the same behaviour (toggling story-panel, it will be hidden once story-panel is hidden)

but I have no issue with keeping the button, so if you advise to get rid of the check above – I'll remove it

Reply:
Maybe we can have something like this when it's closed (in both modes):

![image](https://user-images.githubusercontent.com/7867954/38236267-4ad69d8a-372c-11e8-8e0f-ed25d686828a.png)

The floating button (with opacity). It will open the stories panel with the header in it. This way we can reuse most of the UI between web and mweb.

Reply:
I assume there is a disadvantage for mobile view with floating button. In case of this floating element you will have to always keep it in mind when writing stories of the components. We would need to add some additional containers/spacing in stories to test the component without visual/touch interference with floating element. In my opinion header is OK and quite simple solution for this.

Reply:
👌

Reply:
@igor-dv I prefer this:
https://user-images.githubusercontent.com/3757759/38176267-ac776684-35f4-11e8-9bd0-2c486937f02e.png

both visually and UX, you should not overlay anything over the preview.